ergonomics/ide.ergonomics/src/org/netbeans/modules/ide/ergonomics/fod/ConfigurationPanel.java

```diff
@@ -188,7 +190,7 @@ public void setInfo(FeatureInfo info, String displayName, Collection<UpdateEleme
         List<String> descriptionsList = new ArrayList<>();
         for (FeatureInfo fi : features) {
             String s = required ? fi.getExtraModulesRequiredText(): fi.getExtraModulesRecommendedText();
-            if (!descriptionsList.contains(s)) {
+            if (nonNull(s) && !descriptionsList.contains(s)) {
```

Comment:
While the check does its job isn't it a little overabstracting a simple `s != null` :-)  ?
if there is an explicit `import static` for the `nonNull`, the preceding `import j.u.Objects` is not needed.

Reply:
I removed unused import. I forgot to remove it.

Reply:
is this resolved ? can we merge ?

Reply:
It is resolved